src/pool/srv_iv.c

```diff
@@ -1017,12 +1025,14 @@ pool_iv_pre_sync(struct ds_iv_entry *entry, struct ds_iv_key *key,
 	if (entry->iv_class->iv_class_id != IV_POOL_MAP)
 		return 0;
 
-	pool = ds_pool_lookup(entry->ns->iv_pool_uuid);
+	rc = ds_pool_lookup(entry->ns->iv_pool_uuid, &pool);
 	if (pool == NULL) {
-		D_DEBUG(DB_TRACE, DF_UUID": pool not found\n",
-			DP_UUID(entry->ns->iv_pool_uuid));
+		D_DEBUG(DB_TRACE, DF_UUID": pool not found: %d\n",
+			DP_UUID(entry->ns->iv_pool_uuid), rc);
+		if (rc == -DER_NONEXIST)
+			rc = 0;
 		/* Return 0 to keep forwarding this sync request. */
```

Comment:
does this comment now belong above the if (rc == -DER_NONEXIST) check?

Reply:
oh, yes.